src/nordjylland_news/utils.py

```diff
@@ -0,0 +1,62 @@
+import jsonlines
```

Comment:
Igen, en modul-docstreng er god stil.
```suggestion
"""Utility functions and classes to be used throughout the project."""

import jsonlines
```